test/python/circuit/test_circuit_load_from_qpy.py

```diff
@@ -473,6 +473,35 @@ def test_custom_gate_with_noop_definition(self):
         self.assertIsInstance(new_circ.data[0][0].definition, QuantumCircuit)
         self.assertIs(new_circ.data[1][0].definition, None)
 
+    def test_custom_gate_with_custom_params(self):
+        """Test that a custom gate with overloaded parameter definitions is serialized with
+        proper definitions. Custom instructions have no validation, thus not required for testing"""
```

Comment:
What do you mean by "custom instructions have no validation, thus not required for testing"?

Reply:
- Again the same reason as above, the default `Instruction.validate_parameter` method had no validation and thus it was left out from the testing